src/App.css

```diff
@@ -1,39 +1,13 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+*,
+body {
```

Comment:
As we can see on the Zeplin design there's a main color on the webpage (like a background), which is a grey color, and on each different section (header, card and inputs) there's another color which is white. 

- Kindly change the color to match with the Zeplin design

Zeplin design

![image](https://github.com/naji-97/sec-book-store/assets/93630700/57b92eaf-c37c-4cad-a001-ea7c752b6560)

Your design

![image](https://github.com/naji-97/sec-book-store/assets/93630700/897e5ce0-0373-4b54-b10e-7efddf66f3a5)